atlas-search/public-api/index.md

```diff
@@ -0,0 +1,101 @@
+# Index documentation
```

Comment:
Maybe we can split it in two documents ( index.md and delete.md)

Reply:
for now, it's fine. we can split when the doc will be growing